src/AuthLDAP.php

```diff
@@ -624,9 +624,12 @@ public function showFormAdvancedConfig()
         echo "</td>";
         echo "</tr>";
 
-        echo "<tr class='tab_bg_1'><td><label for='timeout'>" . __('Timeout') . "</label></td>";
-        echo "<td colspan='3'>";
-
+        echo "<tr class='tab_bg_1'>";
+        echo "<td>" . __('TLS Version') . "</td><td>";
+        Dropdown::showFromArray('tls_version', ['1.0', '1.1', '1.2', '1.3'], ['value' => $this->fields["tls_version"]]);
```

Comment:
Maybe a `'' => Dropdown::EMPTY_VALUE` should also be added on top of the proposed versions.